fdbclient/BlobStore.actor.cpp

```diff
@@ -335,58 +350,55 @@ ACTOR Future<Void> deleteRecursively_impl(Reference<BlobStoreEndpoint> b, std::s
 		loop {
 			choose {
 				// Throw if done throws, otherwise don't stop until end_of_stream
-				when(wait(done)) {
-					done = Never();
-				}
+				when(wait(done)) { done = Never(); }
```

Comment:
`when` is not a standard C++, but it is semantically a control statement in Flow. I wonder if there is a way to have a whitespace after custom keywords like `when`, so that it stays consistent with `if`, `else`, `for`.